install_config/configuring_authentication.adoc

```diff
@@ -1391,3 +1391,39 @@ a secret with 32 or 64 bytes.
 <3> Encrypting secret, used to encrypt sessions. Must be 16, 24, or 32
 characters long, to select AES-128, AES-192, or AES-256.
 ====
+
+[[configuring-authentication-security]]
+== Security
+
+Administrators can prevent clients from accessing the API by their `User-Agent`
+header in the new `*userAgentMatching*` configuration setting.
+
+For example, you can deny clients using particular libraries or binaries. This
+example denies the Kubernetes 1.2 client binary and OpenShift Origin 1.1.3
+binary:
+
+====
+----
+userAgentMatchingConfig:
```

Comment:
userAgentMatchingConfig is sub-level under policyConfig. So better to show out "policyConfig:" as well? Otherwise user may not know it.